R/brexit-map.R

```diff
@@ -73,7 +104,7 @@ pmatches = cbind(as.character(las$geo_label[ams]),
 las$geo_label[grep("and", las$geo_label)]
```

Comment:
Ah I thought purrr/you could help with the fuzzy matching fixes - that's the biggest weakness in my analysis so far...